### PR TITLE
Task209 stance detection

### PR DESCRIPTION
**Pull request by Maitreyapatel**
I have created task .json file for stance detection.

### CODE REVIEW

File: tasks/task209_StanceDetection_classification.json [hunk 1]
```diff
@@ -0,0 +1,9219 @@
+{
+    "Contributors": [
+        "Maitreya Patel"
+    ],
+    "Source": [
+        "Unsupervised Stance Detection for Arguments from Consequences by Kobbe et. al."
```
Comment (danyaljj) [inline tasks/task209_StanceDetection_classification.json]: could you include a url to the data source?

File: tasks/task209_StanceDetection_classification.json [hunk 2]
```diff
@@ -0,0 +1,9219 @@
...
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "Given the Target and Argument texts detect the stance that the argument has towards the topic. There are three types of stances \"in favor\", \"against\", and \"neither\". ",
```
Comment (danyaljj) [inline tasks/task209_StanceDetection_classification.json]: Would it make sense to change "neither" to "neutral"?

**Comment by danyaljj** [comment]
Few suggestions; otherwise, it looks good to me!

**Comment by danyaljj** [comment]
Thanks! Looks good to me!

**Review by swarooprm — APPROVED** [review]
Looks good! 
One comment: See if you can create other tasks from the same dataset. The data creation process in the paper may help you in this direction.

**Pull request merged**